src/styles/blocks/days.scss

```diff
@@ -0,0 +1,44 @@
+.days {
+  padding: 10px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+
+  &__day {
+    @include square($days-size);
+    border: 1px solid #000;
+    background-color: #eee;
+    transition: $transition;
+
+    &:hover {
+      background-color: pink;
+      transform: translateY(-20px);
+      cursor: pointer;
+    }
+
+    @for $index from 1 through 31 {
+      &--#{$index}::before {
+        content: "#{$index}";
+        height: 100%;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        font-size: 30px;
```

Comment:
Don't add constant style rules in the loop, the resulting css file will be enormous.

Reply:
Fixed

Reply:
Or maybe my solution didn't fix the constant style, let me know what you meant by constant style rules.

Reply:
The rules which value dorsn't change with each loop iteration.
like display, font-size, height are same for all boxes.